install.sh

```diff
@@ -40,7 +40,7 @@ nvm_source() {
   echo "$NVM_SOURCE_URL"
 }
 
-# 
```

Comment:
Just realised this whitespace got removed by Atom, do you want me to split this out to a separate commit?